### PR TITLE
survey_editor_enhancement_sandbox_sidebar_fix

### PR DESCRIPTION
**Pull request by RusPatel1996**
0. /sandbox1,2,3,4,5 and /surveyeditor1,2,3,4,5 routes are created. Sandbox survey models, views, and migrations are created.

1. Implemented get and put call in sandbox survey view to connect to database.

2. Created and connected up the save button in surveyEditor

3. SurveyEditor is initialized based on the JSON in the database.

4. Sandbox pages are updated based on the JSON in the database.

5.  Issues with sandbox sidebar loading on after survey render is fixed.

6. Copy Link button is implemented in SurveyEditor.

### CODE REVIEW

File: api/api/views/sandbox_survey_view.py [hunk 1]
```diff
@@ -1,12 +1,51 @@
...
-        
-    def put(self, request, pk, name=None):
-        "hello"
-    
+    # permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request):
+        name = request.GET["sandbox_name"]
+        sandbox = SandboxSurvey.objects.get(sandbox_name=name)
+        data = {
```
Comment (seeker25) [inline api/api/views/sandbox_survey_view.py:15]: Try using a serializer for this: 
https://www.django-rest-framework.org/api-guide/serializers/

File: api/api/views/sandbox_survey_view.py [hunk 2]
```diff
@@ -1,12 +1,51 @@
...
 class SandboxSurveyView(generics.GenericAPIView):
-    permission_classes = (permissions.IsAuthenticated,)
-
-    def get(self, request, name=None):
-        "hello"
-        
-    def put(self, request, pk, name=None):
-        "hello"
-    
+    # permission_classes = (permissions.IsAuthenticated,)
```
Comment (seeker25) [inline api/api/views/sandbox_survey_view.py:10]: Shouldn't be making it into master/main with this commented out.

File: api/api/views/sandbox_survey_view.py (hunk 1 above)
Comment (seeker25) [reply api/api/views/sandbox_survey_view.py:15]: https://github.com/bcgov/representation-grant-app/blob/main/api/api/serializers.py

File: api/api/views/sandbox_survey_view.py (hunk 2 above)
Comment (RusPatel1996) [reply api/api/views/sandbox_survey_view.py:10]: 😨 Oops yep definitely forgot to re-enable that. Good catch!

**Pull request merged**